scripts/deploy-patron-kyc.js

```diff
@@ -38,13 +38,13 @@ const initMessage =
  Please verify them carefully..\n";
 
 const initializeContract = async (_deployedContract) => {
-  const start = Math.floor(new Date().getTime() / 1000) + 1 * 60;
-  const end = start + 24 * 3600;
+  const start = process.env.START_DATE;
+  const end = process.env.END_DATE;
 
-  const ratio = ethers.utils.parseUnits("0.004", 18);
-  const hardCap = ethers.utils.parseUnits("10", "ether");
-  const contributionLimit = ethers.utils.parseUnits("0.5", "ether");
-  const patronRoles = [ethers.utils.namehash("email.roles.verification.apps.energyweb.iam.ewc")];
+  const ratio = ethers.utils.parseUnits(process.env.RATIO, 18);
+  const hardCap = ethers.utils.parseUnits(process.env.HARD_CAP, "ether");
+  const contributionLimit = ethers.utils.parseUnits(process.env.CONTRIBUTION_LIMIT, "ether");
+  const patronRoles = [ethers.utils.namehash(process.env.PATRON_ROLES)];
```

Comment:
ROLE needs to be singular for this to work, I believe
```suggestion
  const patronRoles = [ethers.utils.namehash(process.env.PATRON_ROLE)];
```

Reply:
oh yes correct. Thank you @jrhender

Reply:
Impelemented on [f9492a5](https://github.com/energywebfoundation/staking-pool/pull/69/commits/f9492a53952e217d052c24499d3b94a584231097)